core/controlplane/config/templates/cloud-config-worker

```diff
@@ -303,12 +304,12 @@ coreos:
         --cloud-provider=aws \
         --cert-dir=/etc/kubernetes/ssl \
         {{- if and .Experimental.TLSBootstrap.Enabled .AssetsConfig.HasTLSBootstrapToken }}
-        --experimental-bootstrap-kubeconfig=/etc/kubernetes/worker-bootstrap-kubeconfig.yaml \
+        --experimental-bootstrap-kubeconfig=/etc/kubernetes/kubeconfig/worker-bootstrap.yaml \
```

Comment:
worker bootstrap actually need SSL certs, at least ca.pem to verify  connection

Reply:
What is your suggestion to fix this? Re-introduce the volume mount to `/etc/kubernetes/ssl` to kube-proxy `DaemonSet`, or somehow putting the `ca.pem` file in another directory in order to mount just this file?

Either way, I think we should deprecate the old way of bootstrapping worker nodes ASAP in favor of the TLS bootstrap feature.